Main svc

DmitriyDemintsev opened the pull request.

DmitriyDemintsev committed:
сохранение реализации без подборок

DmitriyDemintsev committed:
версия реализации с прохождением postman до прогона docker

DmitriyDemintsev committed:
выполнена реализация основного сервиса

DmitriyDemintsev committed:
поправлены pom-файлы по итогам прогона в гитхабе

DmitriyDemintsev committed:
исправлены замечания check and build application

mikita95 commented on ewm-service/pom.xml:
```diff
@@ -0,0 +1,138 @@
+        <dependency>
+            <groupId>org.example</groupId>
+            <artifactId>stats-client</artifactId>
+            <version>0.0.1-SNAPSHOT</version>
```
Номера версий лучше вынести свойства pom файла, так будет удобнее ими управлять

mikita95 commented on ewm-service/src/main/java/ru/practicum/dto/event/EventShortDto.java:
```diff
@@ -0,0 +1,28 @@
+    private String title;
+    private Integer confirmedRequests;
+    private Integer views;
+//    private Long comments;  // для фичи
```
Закомментированного кода быть не должно :)

mikita95 commented on ewm-service/src/main/java/ru/practicum/model/Event.java:
```diff
@@ -0,0 +1,63 @@
+import java.time.LocalDateTime;
+
+
+@Data
```
Аннотацию Data нужно аккуратно использовать над сущностями hibernate, так как это сгенерирует методы toString и hashCode, которые будут включать все поля, что может привести к циклическим вызовам, либо к инициализации lazy полей

mikita95 reviewed (changes requested):
Все сделано отлично :) Оставил лишь небольшие замечания

DmitriyDemintsev committed:
исправлены замечания в отношении:
- вынесения номеров версий в свойства pom-файла
- закомментированного кода
- аннотации Data

mikita95 reviewed (approved):
Все отлично:) Можно мержить пул-реквест
